# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1310919143 -7200
#      Sun Jul 17 18:12:23 2011 +0200
# Node ID 751bbbcfd6e367240df0ac7b6d2463049d81485f
# Parent  86d6c3b90ad7fdc39d9842dc8e39d45d14d4ff80
Renamed project files and target binary to octave-gui.

diff --git a/gui/Quint.pro b/gui/octave-gui.pro
rename from gui/Quint.pro
rename to gui/octave-gui.pro
--- a/gui/Quint.pro
+++ b/gui/octave-gui.pro
@@ -14,17 +14,17 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 
 # Basic settings:
 QT                  += core gui webkit xml opengl   # Qt modules
 TEMPLATE            = app                           # Build as application
-TARGET              = Quint                         # Name of the target binary
+TARGET              = octave-gui                    # Name of the target binary
 
 DESTDIR             = bin                           # Destination of the output
 UI_DIR              = ui-files                      # Folder for ui files
 MOC_DIR             = moc-files                     # Folder for moc files
 OBJECTS_DIR         = object-files                  # Folder for object files
 
 TRANSLATIONS        += languages/german             # Available translations
 LIBS                += -lqscintilla2
diff --git a/gui/start.sh b/gui/start.sh
--- a/gui/start.sh
+++ b/gui/start.sh
@@ -12,9 +12,9 @@
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 LANG=en_US.UTF-8
 cd bin
-./Quint
+./octave-gui
